documentation/projectVision.md

```diff
@@ -0,0 +1,20 @@
+# VISION
+ Help development teams save their projects from unexpected assembly breakdowns and security issues due to vulnerabilities in third party libraries. 
+# TARGET GROUP
+- Software developers, project managers;
+- Security/code auditors.
+# OBJECTIVES AND GOALS
+- Fast and easily find vulnerabilities in - user’s projects;
+- Have an ability to see statistical data for found vulnerabilities.
+# FEATURES
+- As a user I want to have an opportunity to register and authorization on website;
+- As a user I want to be able to create projects on my account;
+- As a user I want to upload zip file with sources into project to analyze them on vulnerabilities;
+- As a user I want to have an opportunity to see results of analyzing uploaded sources;
+- As a user I want to be able to see user information of other people’s;
+- As a user I want to be able to see statistic of uploaded zip files and their vulnerabilities;
```

Comment:
Let's change to "I want to be able to scan project several times and see statistical data"